docs/user-guide/resource_hooks.md

```diff
@@ -12,7 +12,7 @@ Kubernetes rolling update strategy.
 
 ## Usage
 
-Hooks are simply Kubernetes manifests inside your Argo CD Application annotated with `argocd.argoproj.io/hook`, e.g.:
+Hooks are simply Kubernetes manifests tracked in the source repository of your Argo CD Application annotated with `argocd.argoproj.io/hook`, e.g.:
```

Comment:
It was unclear to me what "inside your Argo CD Application" actually meant. I'm still not sure this is 100% right, but I think something like this would have saved me lots of time.